lib/tools/adb-emu-commands.js

```diff
@@ -18,29 +18,37 @@ emuMethods.isEmulatorConnected = async function (udid) {
   return false;
 };
 
+emuMethods.checkEmulatorConnected = async function (udid) {
```

Comment:
based on how I read the logic here, if you fail to pass in a `udid` to this method, this method will do precisely nothing. is that intended? maybe we should error instead?